app/src/main/java/com/jjswigut/matters/database/MatterDataBaseDao.kt

```diff
@@ -0,0 +1,31 @@
+
+
+package com.jjswigut.matters.database
+
+import androidx.lifecycle.LiveData
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.Query
+
+@Dao
+interface MatterDataBaseDao{
+
+    @Insert
+    suspend fun insert(matter: Matter)
+
+    @Insert
+    suspend fun update(matter: Matter)
+
+    @Query("SELECT * from matter_table WHERE matterId = :key")
+    suspend fun get(key: Long): Matter?
+
+    @Query("DELETE FROM matter_table")
```

Comment:
I believe room gives us a `Delete` annotation?

Reply:
So I tried to change this but it said that I needed parameters. I looked at the documentation for Room but wasn't really sure what I was looking for?

Reply:
ah, I had originally misread. you're looking to delete the entire table and not just an entry from the table - my bad!